requirements.txt

```diff
@@ -0,0 +1,29 @@
+#
+# This file is autogenerated by pip-compile with python 3.9
```

Comment:
make sure the requirements are the same as in web server when applicable

Reply:
I used the requirements folder (copied from the webserver ) in dockerfile.